.travis.yml

```diff
@@ -77,3 +78,4 @@ install:
 script:
   - industrial_ci/_wrap_test.sh # this script is only for internal use
   - if ! [ -z "$POST_PROCESS" ]; then $POST_PROCESS; fi
+  - if [ -n "$DOCKER_COMMIT" ]; then echo "Test if the Docker image is committed or not. See the size of the Docker image is non-zero or not."; docker image inspect "$DOCKER_COMMIT" --format="$DOCKER_COMMIT: {{.Size}} bytes"; fi
```

Comment:
The echo'd text does not make sense anymore.
It might be better to just set
```
POST_PROCESS=docker image inspect "$DOCKER_COMMIT" --format="$DOCKER_COMMIT: {{.Size}} bytes"
```
And line ending is missing